lib/server/push.api.js

```diff
@@ -411,10 +411,16 @@ function topicsSend(topics, raixNotification) {
 
   console.log('Push: Sending notification to topic(s): ', raixNotification);
 
+  const appIdentifier = raixNotification.appIdentifier;
   topics.forEach(function (topic) {
-    const fcmMessage = getFcmMessageForTopic(topic, raixNotification, raixNotification.topicsPlatform[topic]);
-    const appIdentifier = raixNotification.appIdentifier;
-    firebasePromises.push(sendFcmMessage(appIdentifier, fcmMessage));
+    if (topic === 'android_users') {
+      firebasePromises.push(sendFcmMessage(appIdentifier, getFcmMessageForTopic(topic, raixNotification, 'android')));
+    } else if (topic === 'ios_users') {
+      firebasePromises.push(sendFcmMessage(appIdentifier, getFcmMessageForTopic(topic, raixNotification, 'ios')));
+    } else {
+      firebasePromises.push(sendFcmMessage(appIdentifier, getFcmMessageForTopic(topic, raixNotification, 'android')));
+      firebasePromises.push(sendFcmMessage(appIdentifier, getFcmMessageForTopic(topic, raixNotification, 'ios')));
```

Comment:
@IlicPavle For the practice that is in the shared app, this will send the same notification to the same app twice. This will happen only on Android I think. We need to revert the usage of `topicsPlatform`, and when subscribing app for a specific practice, we would need to subscribe them not to practice identifier, but for a combination  of practice identifier and platform, for example, `practiceIdOrIdentifier_platform`.